### PR TITLE
chore: Remove addCssBlock method from generated file

### PR DESCRIPTION
**Pull request by Artur-**

### CODE REVIEW

File: flow-server/src/main/java/com/vaadin/flow/server/frontend/TaskUpdateImports.java [hunk 1]
```diff
@@ -211,7 +186,7 @@ protected String getImportsNotFoundMessage() {
 
         protected List<String> getDefinitionLines() {
             List<String> lines = new ArrayList<>();
-            addLines(lines, EXPORT_MODULES_DEF);
+            addLines(lines, "export { }");
             return lines;
```
[the remark this answers is not in the document]
Comment (Artur-) [reply flow-server/src/main/java/com/vaadin/flow/server/frontend/TaskUpdateImports.java:190]: The file needs to exist and there needs to be an empty export for TS to be happy..

File: flow-plugins/flow-maven-plugin/src/test/java/com/vaadin/flow/plugin/maven/BuildFrontendMojoTest.java [hunk 2]
```diff
@@ -346,14 +346,7 @@ public void changedBuildDirectory_resourcesCopiedNoTargetFolderExists()
...
 
-        List<String> expectedLines = new ArrayList<>(Arrays.asList(
-                "export const addCssBlock = function(block, before = false) {",
-                " const tpl = document.createElement('template');",
-                " tpl.innerHTML = block;",
-                " document.head[before ? 'insertBefore' : 'appendChild'](tpl.content, document.head.firstChild);",
-                "};",
-                "addCssBlock('<custom-style><style include=\"lumo-color lumo-typography\"></style></custom-style>', true);"));
-
+        List<String> expectedLines = new ArrayList<>();
```
[the remark this answers is not in the document]
Comment (Artur-) [reply flow-plugins/flow-maven-plugin/src/test/java/com/vaadin/flow/plugin/maven/BuildFrontendMojoTest.java:349]: ✅

File: flow-server/src/main/java/com/vaadin/flow/server/frontend/TaskUpdateImports.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (Artur-) [reply flow-server/src/main/java/com/vaadin/flow/server/frontend/TaskUpdateImports.java:190]: ✅

File: flow-server/src/main/java/com/vaadin/flow/server/frontend/TaskUpdateImports.java [hunk 3]
```diff
@@ -135,28 +132,6 @@ protected void writeImportLines(List<String> lines) {
         @Override
         protected Collection<String> getThemeLines() {
             Collection<String> lines = new ArrayList<>();
```
[the remark this answers is not in the document]
Comment (Artur-) [reply flow-server/src/main/java/com/vaadin/flow/server/frontend/TaskUpdateImports.java]: ✅

**Review by caalador — APPROVED** [review]

File: flow-server/src/test/java/com/vaadin/flow/server/frontend/NodeUpdateImportsTest.java [hunk 4]
```diff
@@ -248,7 +237,7 @@ Logger log() {
                 importsDefinitionFile, Charset.defaultCharset());
 
         MatcherAssert.assertThat(definitionContent, CoreMatchers.containsString(
-                "export declare const addCssBlock: (block: string, before?: boolean) => void;"));
+                "export { }"));
```
Comment (Artur-) [inline flow-server/src/test/java/com/vaadin/flow/server/frontend/NodeUpdateImportsTest.java:240]: Added this part of the test back as the definition file still needs to be there

**Review by caalador — APPROVED** [review]

**Review by caalador — APPROVED** [review]

**Pull request merged**